src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs

```diff
@@ -82,6 +83,14 @@ public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavio
             PromptBehavior = promptBehavior;
         }
 
+        /// <summary>
+        /// Constructor that allows extends to configure their own web ui. Not supported on Android, iOS and UWP. 
+        /// </summary>
+        public PlatformParameters(PromptBehavior promptBehavior, ICustomWebUi customWebUi)
```

Comment:
As discussed with @jmprieur , we will expose this extensibility point via PlatformParamters. I have added this ctor to all platforms and on Android, iOS and UWP I am throwing a PlatformNotSupported. This is to avoid having MissingMethodException, because for ADAL we do not have a pure NetCore impl, so users targetting XForms, for example, will see this ctor.